plugins/chat/assets/javascripts/discourse/components/chat/routes/channel-info-members.gjs

```diff
@@ -168,17 +190,28 @@ export default class ChatRouteChannelInfoMembers extends Component {
             {{#each this.members as |membership|}}
               <li
                 class="c-channel-members__list-item -member"
-                {{on "click" (fn this.openMemberCard membership.user)}}
-                {{this.onEnter (fn this.openMemberCard membership.user)}}
                 tabindex="0"
               >
-                <ChatUserInfo
-                  @user={{membership.user}}
-                  @avatarSize="tiny"
-                  @interactive={{false}}
-                  @showStatus={{true}}
-                  @showStatusDescription={{true}}
-                />
+                <a
```

Comment:
Had to re-arrange a but to create two clickable areas.

**Before:**

```
- li (click opens user card)
  - ChatUserInfo
```

**After:**

```
- li
  - a (click opens user card)
    - ChatUserInfo
  - DButton (click removes user)
```

The user info section grows to consume the available clickable area:

```
--------------------------------------
| ChatUserInfo          --> | Remove |
--------------------------------------
```